clients/unshaded/src/main/java/tachyon/client/file/options/CreateOptions.java

```diff
@@ -143,6 +143,22 @@ public UnderStorageType getUnderStorageType() {
   }
 
   /**
+   * @return the name : value pairs for all the fields
+   */
+  @Override
+  public String toString() {
+    return "CreateOptions{mBlockSizeBytes=" + mBlockSizeBytes + ", mRecursive=" + mRecursive
+        + ", mTTL=" + mTTL + ", mUnderStorageType=" + mUnderStorageType.toString() + "}";
+  }
+
+  /**
+   * @return the default String representation for testing purposes
```

Comment:
I think it is unnecessary to add these methods for the sake of unit tests.

If the tests want a string representation of the object with no spaces, they can do: `op.toString().st.replaceAll("\\s+","")` and the tests don't care about the string being meaninful, they can use `op.hashCode()`